vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -129,7 +136,10 @@ def fused_moe_kernel(
                     mask=offs_k[:, None] < K - k * BLOCK_SIZE_K,
                     other=0.0)
         # We accumulate along the K dimension.
-        accumulator += tl.dot(a, b)
+        if use_fp8:
+            accumulator = tl.dot(a, b, acc=accumulator, allow_tf32=True)
```

Comment:
nit: allow_tf is [deprecated](https://github.com/openai/triton/blob/7c6c2ef26430d34bfec657d6e2d957505425e577/python/triton/language/core.py#L1512) and it is recommended to use `input_precision="tf32"`
Also it might be worth noting if this is either: required for fp8 to work or just a performance optimization

Reply:
So the `input_precision="tf32"` interface is not part of the triton 2.2.0 release yet (which is the version we are using -- triton is pinned to 2.2.0 via the pytorch version we are pinning).

I tried without this flag and can't measure a performance difference, so will remove it. It seems it is the default for devices with tensor cores anyways (https://triton-lang.org/main/python-api/generated/triton.language.dot.html#triton.language.dot). Removing the flag will make us robust against the `allow_tf32` vs. `input_precision` interface change as well. Once we have upgraded trition, it might be worth looking at https://github.com/openai/triton/pull/3234